ante/fee-notes.md

```diff
@@ -0,0 +1,53 @@
+# Notes about Gas prices and gas used
+
+NOTES:
+
+- the gas used depends on the current state of the chain
+- the gas records below are based on the empty state
+- user is charged the gas-limit, even if tx consumed less gas
+
+| operation                                      | gas used |
+| :--------------------------------------------- | -------: |
+| x/bank send                                    |   31'029 |
+| x/group create                                 |   68'908 |
+| x/oracle MsgAggregateExchangeRateVote (3 curr) |   66'251 |
+| x/oracle MsgAggregateExchangeRateVote (6 curr) |   69'726 |
+| default gas limit                              |  200'000 |
+
+## Target price (in USD cent) for x/bank send
+
+| umee price (usd cent) | gas price in uumee | fee (usd cent) |
+| --------------------: | -----------------: | -------------: |
+|                     2 |                0.2 |         0.0124 |
+|                     2 |                0.1 |         0.0062 |
+|                     2 |               0.02 |        0.00124 |
+|                     5 |                0.2 |          0.031 |
+|                     5 |                0.1 |         0.0155 |
+|                     5 |               0.02 |         0.0031 |
+
+## Target price (in USD cent) for validator oracle txs (with 6 currencies) per day
+
+There are roughly 10tx / minute and 14400 per day.
+Validator has to do 2 tx (prevote and vote) every 5 blocks.
```

Comment:
oracle prevote / vote is free :thinking:

Reply:
yes, but the gas is still consumed.

Reply:
oh, you were asking about the table? This is indicative. It says "if we were to charge x amount of gas then this will be the cost". Do you think I should make it more clear?